spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -238,5 +238,36 @@ def expect_retryable_error(error_class)
         expect(Form526JobStatus.last.status).to eq Form526JobStatus::STATUS[:non_retryable_error]
       end
     end
+
+    context 'with an RRD claim' do
```

Comment:
These new tests are copied from earlier tests. The difference is `rrd_applicable?` returns true so that we can `expect_any_instance_of(described_class).to receive(:send_rrd_alert)`.